src/build.py

```diff
@@ -161,8 +162,11 @@ def NodePlatformName():
 
 # Known failures.
 IT_IS_KNOWN = 'known_gcc_test_failures.txt'
-LLVM_KNOWN_TORTURE_FAILURES = [os.path.join(LLVM_SRC_DIR, 'lib', 'Target',
-                                            'WebAssembly', IT_IS_KNOWN)]
+LLVM_KNOWN_TORTURE_FAILURES_S = [os.path.join(LLVM_SRC_DIR, 'lib', 'Target',
+                                              'WebAssembly', IT_IS_KNOWN)]
+# TODO(sbc): Move this failures file to the llvm repo
```

Comment:
Its counterpart (the known compile failures with -s) is in the LLVM repo, in the wasm backend directory; they should at least be together. The idea behind putting them in the LLVM repo was that you could fix a compile failure and update the expectations in a single commit. [edit: but see below, we probably don't need a separate file]